### PR TITLE
Add support for Microsoft Surface RT

### LINKED ISSUE #53: Support for Microsoft Surface RT Tablet
Hi, can you add support for the Microsoft Surface RT Tablet? [It supports the RPi OS](https://openrt.gitbook.io/open-surfacert/surface-rt/linux/root-filesystem/distros/raspberry-pi-os), and your software works with some tweaks.

The backlight is controlled by `/sys/class/backlight/backlight` (instead of `/sys/class/backlight/rpi_backlight`).

I edited that in `__init.py__` and it works. However, it would be great if this is included in this software.

### PR DESCRIPTION
**Pull request by apandada1**
Added Surface RT support with automatic detection. Fixes #53 

Tested on my Surface RT.

### CODE REVIEW

**Comment by apandada1** [comment]
Please release a new version (so that Surface users can easily install with pip) when merged.

File: rpi_backlight/__init__.py [hunk 1]
```diff
@@ -75,7 +78,10 @@ def __init__(
         self._board_type = board_type
         self._fade_duration = 0.0  # in seconds
 
-        if self._board_type == BoardType.RASPBERRY_PI:
+        if (
+            self._board_type == BoardType.RASPBERRY_PI
+            or self._board_type == BoardType.MICROSOFT_SURFACE_RT
+        ):
```
Comment (linusg) [inline rpi_backlight/__init__.py:84]:
```suggestion
        if self._board_type in BoardType.RASPBERRY_P,I BoardType.MICROSOFT_SURFACE_RT):
```

File: README.md [hunk 2]
```diff
@@ -28,7 +28,7 @@
 ## Requirements
 
 - A **Raspberry Pi or ASUS Tinker Board** including a correctly assembled **7" touch display v1.1 or higher**
-  (look on the display's circuit board to see its version) running a Linux-based OS
+  (look on the display's circuit board to see its version), or as [Surface RT](https://openrt.gitbook.io/open-surfacert/surface-rt/linux/root-filesystem/distros/raspberry-pi-os) running a Linux-based OS
```
Comment (linusg) [inline README.md:31]:
```suggestion
  (look on the display's circuit board to see its version), or a [**Surface RT**](https://openrt.gitbook.io/open-surfacert/surface-rt/linux/root-filesystem/distros/raspberry-pi-os) running a Linux-based OS
```

**Review by linusg — CHANGES_REQUESTED** [review]

**Comment by apandada1** [comment]
Done :)

**Comment by apandada1** [comment]
Any updates? I am not sure why the GitHub build failed. The software seems to work without an issue.

**Comment by linusg** [comment]
you didn't run black, click on "Details" next to the red X

**Comment by apandada1** [comment]
Oh, I didn't know about black before. Finally, done :)

**Comment by apandada1** [comment]
Can you please release a new version and push it to pypi?

**Pull request merged**